Type error in initializers/db.ts (TS2353) when consumers run tsc

## Problem

When a Keryx consumer project runs `tsc --noEmit` (or `tsc` with `noEmit: true` in tsconfig), TypeScript follows imports into `node_modules/keryx/initializers/db.ts` and reports:

```
node_modules/keryx/initializers/db.ts(118,9): error TS2353: Object literal may only specify known properties, and 'url' does not exist in type '{ wranglerConfigPath: string; dbName: string; }'.
```

Because keryx distributes raw `.ts` source files (not `.d.ts` declarations), `skipLibCheck: true` in the consumer's tsconfig has no effect — that flag only skips `.d.ts` files.

## Reproduction

1. `bunx keryx new my-app --yes`
2. `cd my-app && bun install`
3. `npx tsc --noEmit`

## Expected

`tsc` should pass cleanly for a freshly-scaffolded project.

## Possible fixes

- Fix the type error in `initializers/db.ts` line 118 (the `url` property doesn't match the expected type)
- Or: ship `.d.ts` declaration files alongside source so `skipLibCheck` works for consumers

--- a/packages/keryx/initializers/db.ts
+++ b/packages/keryx/initializers/db.ts
@@ -111,14 +111,14 @@ export class DB extends Initializer {
    * Learn more @ https://orm.drizzle.team/kit-docs/overview
    */
   async generateMigrations() {
-    const migrationConfig = {
-      dialect: "postgresql",
+    const migrationConfig: DrizzleMigrateConfig = {
+      dialect: "postgresql" as const,
       schema: path.join("schema", "*"),
       dbCredentials: {
         url: config.database.connectionString,
       },
       out: path.join("drizzle"),
-    } satisfies DrizzleMigrateConfig;
+    };
 
     const fileContent = `export default ${JSON.stringify(migrationConfig, null, 2)}`;
     const tmpfilePath = path.join(api.rootDir, "drizzle", "config.tmp.ts");

--- a/packages/keryx/package.json
+++ b/packages/keryx/package.json
@@ -1,6 +1,6 @@
 {
   "name": "keryx",
-  "version": "0.10.4",
+  "version": "0.10.5",
   "module": "index.ts",
   "type": "module",
   "license": "MIT",

